client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/special-instructions.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Icon } from '@wordpress/components';
+import { chevronDown, chevronUp } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { Controller, Control } from 'react-hook-form';
+import FormTextArea from 'calypso/components/forms/form-textarea';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { CredentialsFormData } from '../types';
+import { ErrorMessage } from './error-message';
+
+interface Props {
+	control: Control< CredentialsFormData >;
+	errors: any;
+}
+
+export const SpecialInstructions: React.FC< Props > = ( { control, errors } ) => {
+	const translate = useTranslate();
+	const [ showNotes, setShowNotes ] = useState( false );
+
+	const toggleShowNotes = () => {
+		setShowNotes( ! showNotes );
+		recordTracksEvent( 'calypso_site_migration_special_instructions_toggle', {
```

Comment:
Maybe not related to this PR.

I am wondering why we want to know if the user is opening/closing the notes. 
Would it be better to check the number of tickets using the notes we have?

Reply:
I haven't done anything to address this, but maybe we can discuss about keeping/discarding it in a different PR

cc: @andres-blanco @valterlorran